build.gradle.kts

```diff
@@ -202,6 +202,7 @@ val javadocAggregateIncludingTests by tasks.registering(Javadoc::class) {
     description = "Generates aggregate javadoc for all the artifacts"
 
     val sourceSets = subprojects
+        .filter { it.name != "bom" }
```

Comment:
`javadocAggregate` section uses `.filter { it.names.contains("main") }`, we'd better to keep them consistent? (Both way looks good to me).

Reply:
I prefer to keep it as is. Thanks for taking a look!